index.html

```diff
@@ -254,10 +257,10 @@ <h3>2015</h3>
   <section class="contact-me">
     <div class="head-section-co">
       <div id="head-container-co">
-        <h1 class="head-co">
-          About, <br />
+        <h2 class="head-co">
+          About <br />
           Myself
```

Comment:
Great job @obibaadoma 🤝

- [ ] [Optional] As stated in the project requirements, kindly align the elements of the contact me section with flexbox.